doc/reference/devices_nic.md

```diff
@@ -451,6 +534,16 @@ Key                     | Type    | Default           | Description
 `queue.tx.length`       | integer | -                 | The transmit queue length for the NIC
 `vlan`                  | integer | -                 | The VLAN ID to attach to
 
+#### Configuration examples
+
+Add a `routed` network device to an instance using `nictype`:
+
+    lxc config device add <instance_name> <device_name> nic nictype=routed
```

Comment:
routed nic type requires at least one ip address, so we should so that in the example